RELEASING.md

```diff
@@ -0,0 +1,54 @@
+Releasing a new version
+=======================
+
+We will use the `v0.5.3` release (#738) as an example of how to release a new
+FDO version:
+
+* Fork the repo and create a new branch for the new release:
+
+    ```bash
+    gh repo fork fdo-rs/fido-device-onboard-rs --clone --remote
+    git pull upstream main
+    git checkout -b prepare-v0.5.3
+    ```
+
+* Update the `fido-device-onboard.spec` file and set the new version: `Version: 0.5.3`
+* Update the `version` within the `[[package]]` section in all the `Cargo.toml` files.
+* Update the `[[dependencies]]` in all the `Cargo.toml` files to use the latest
+FDO versions.
+* Update the `libfdo-data/fdo_data.h` file to reflect the correct version:
+
+    ```c
+    #define FDO_DATA_MAJOR 0
+    #define FDO_DATA_MINOR 5
+    #define FDO_DATA_PATCH 3
+    ```
+
+* Update `Cargo.lock` file:
+
+    ```bash
+    cargo update --offline --workspace
+    ```
+
+* Commit all the changes and create a PR (see #738 with all the changes described
+above):
+
+    ```bash
+    git add *
```

Comment:
Just a minor, but in an update of Cargo.toml and bumping versions shouldn't need to add new files, explicitly doing an add here may add unrelated files required to bump a release so drop that, the rest LGTM.

Reply:
I added the exact files changed in the example release (#738) to be consistent with the  rest of the document